modules/postgresql/main.tf

```diff
@@ -320,15 +325,62 @@ resource "google_sql_user" "iam_account" {
   deletion_policy = var.user_deletion_policy
 }
 
+
+resource "google_compute_address" "psc_ilb_consumer_address" {
```

Comment:
For each resource that supports project argument let's explicitly provide it and not depend on inheriting from provider.